compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -646,8 +646,15 @@ public void visit(BLangListConstructorExpr listConstructor) {
                             // TODO: 8/21/19 The fix below is a temporary one. The rest type should be NoType if
```

Comment:
This comments no longer apply, isn't it ?

Reply:
Removed in 000740f